orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DisableClusterTask.groovy

```diff
@@ -36,6 +36,16 @@ class DisableClusterTask extends AbstractClusterWideClouddriverTask {
     boolean preferLargerOverNewer = false
   }
 
+  public static class DisableOperation {
+    Integer desiredPercentage
```

Comment:
Yup I extended all the enable/disable descriptions a little while ago: https://github.com/spinnaker/clouddriver/commit/cc10d6f8cb8191ed72294fc751b4cf520476b124